index.d.ts

```diff
@@ -0,0 +1,124 @@
+  export interface BitmapTextProperties extends ContainerWithChildren<PIXI.extras.BitmapText> {
+    text: string;
+  }
+
+  /**
+   * A component wrapper for `PIXI.extras.BitmapText`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.extras.BitmapText.html
+   */
+  export class BitmapText extends React.Component<BitmapTextProperties> {}
+
+  /** `Container` component properties. */
+  export interface ContainerProperties extends ContainerWithChildren<PIXI.Container> {}
+
+  /**
+   * A component wrapper for `PIXI.extras.BitmapText`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.Container.html
+   */
+  export class Container extends React.Component<ContainerProperties> {}
+
+  /** `Graphics` component properties. */
+  export interface GraphicsProperties extends ContainerWithChildren<PIXI.Graphics> {}
+
+  /**
+   * A component wrapper for `PIXI.Graphics`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.Graphics.html
+   */
+  export class Graphics extends React.Component<GraphicsProperties> {}
+
+  /** `ParticleContainer` component properties. */
+  export interface ParticleContainerProperties extends ContainerWithChildren<PIXI.particles.ParticleContainer> {}
+
+  /**
+   * A component wrapper for `PIXI.particles.ParticleContainer`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.particles.ParticleContainer.html
+   */
+  export class ParticleContainer extends React.Component<TilingSpriteProperties> {}
+
+  /** `Sprite` component properties. */
+  export interface SpriteProperties extends ContainerWithChildren<PIXI.Sprite> {}
+
+  /**
+   * A component wrapper for `PIXI.Sprite`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.Sprite.html
+   */
+  export class Sprite extends React.Component<SpriteProperties> {}
+
+  /** `Text` component properties */
+  export interface TextProperties extends ContainerWithChildren<PIXI.Text> {}
+
+  /**
+   * A component wrapper for `PIXI.Text`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.Text.html
+   */
+  export class Text extends React.Component<TextProperties> {}
+
+  /** `TilingSprite` component properties. */
+  export interface TilingSpriteProperties extends ContainerWithChildren<PIXI.extras.TilingSprite> {
+    texture: PIXI.Texture;
+  }
+
+  /**
+   * A component wrapper for `PIXI.extras.TilingSprite`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.extras.TilingSprite.html
+   */
+  export class TilingSprite extends React.Component<TilingSpriteProperties> {}
+
+  /** `Stage` component properties." */
+  export interface StageProperties extends ContainerWithChildren<PIXI.Container> {}
+
+  /**
+   * A component wrapper for `PIXI.Application`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.Application.html
+   */
+  export class Stage extends React.Component<StageProperties> {}
+
+  /** Custom React Reconciler render method. */
+  export function render(pixiElement: any, stage: any, callback?: Function): void;
```

Comment:
Are you sure `any` is a good type here?

Reply:
The real type definition for this should come from `react-reconciler` but that does not currently have type definitions.  Given that, and that ostensibly consumers don't use this method, I think `any` fine.  If you happen to know what these values are sufficiently to narrow them down I would be happy to adjust them.

Reply:
Fair enough

Reply:
I've improved these definitions.